README.md

```diff
@@ -1,7 +1,42 @@
 # Babel Plugin: Annotate React
 ​
-This repo will hold a Babel plugin that annotates React components with two attributes: `data-component` and `data-element`. These attribues will allow you to use selectors to more accurately target React-rendered components when testing and when emitting Fullstory recording events.
-​
-This plugin will work with both web and native React applications.  
-​
-* Status: This is a work-in-progress and is not yet ready for production use. *
+This repo will hold a Babel plugin that annotates React components with stable attributes that can be used to search and select using FullStory. This is most useful when using a React system that generates dynamic names for Components.
+
+For React on the web the attributes are `data-component` and `data-element`.
+
+Example input:
+
+	class HelloComponent extends Component {
```

Comment:
At the risk of opening up the "tabs vs. spaces" debate: how would you feel about using spaces for indentation, 2 spaces per indent?